.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "javascript.preferences.quoteStyle": "double",
+  "exportall.config.quote": "\"",
+  "exportall.config.relExclusion": [
+    "/lib/modules/shared/exports.ts",
+    "/lib/modules/shared/types/exports.ts"
+  ]
+}
```

Comment:
Is it normal for this file to be there?

Reply:
Yes, It's useful to share basic VSCode settings.
Especially on certain repositories like this one which still have a specific configuration like double quote style instead of simple quote.

Maybe we should rationalize these types of configurations between project but that need a huge PR to reformat all code, to accepted style. So for the moment IMHO, it's the better solution to share these settings for contributors